src/app/app.component.html

```diff
@@ -192,26 +194,64 @@
                       </span>
                     </div>
                     <ui5-bar design="Footer">
-                      <ui5-button design="Emphasized" slot="endContent">{{'UPGRADE' | ui5I18n}}</ui5-button>
+                      <ui5-button design="Emphasized" slot="endContent" (click)="openUpgrade()" [disabled]="openCheckIn" >{{'UPGRADE' | ui5I18n}} </ui5-button>
                     </ui5-bar>
                   </ui5-card>
                 </div>
               </div>
+              
+              <!-- Baggage section -->
+              <div [fdLayoutGridCol]="12" [colMd]="12" [colLg]="12" [colXl]="12" 
+              class="remove-padding cards__column">
+              
+                  <div [fdLayoutGridCol]="12" [colMd]="6" [colLg]="6" [colXl]="7" class="remove-padding">
+                    <div class="cards__row baggage--allowance">
+                      <ui5-card>
+                        <ui5-card-header slot="header" [title-text]="'Baggage Allowance' | ui5I18n">
+                        </ui5-card-header>
+                        <app-baggage-allowance></app-baggage-allowance>
+                        
+                      </ui5-card>
+                    </div>
+                  </div>
+         
 
-              <!-- Payment Details section -->
-              <div [fdLayoutGridCol]="12" [colMd]="6" [colLg]="12" [colXl]="12" class="remove-padding">
-                <div class="payment-details-card">
-                  <ui5-card class="card--wide">
-                    <ui5-card-header slot="header" [title-text]="'PAYMENT_DETAILS' | ui5I18n"></ui5-card-header>
-                    <app-payment-details></app-payment-details>
-                  </ui5-card>
-                </div>
+           <!-- Explore section -->
+           
+      
+              
+  
+           
```

Comment:
You do not need this much empty lines between tags. This comment applies to every other HTML file here